### ISSUE TITLE
[Bug report] plugin-pwa - disabling viewport scaling violates WCAG accessibility standards

### ISSUE THREAD

**Issue by nruffing** [vuepress/ecosystem#84]
### Description

`plugin-pwa` replaces the `viewport` meta element forces `user-scalable=no` to be set.
https://github.com/vuepress/ecosystem/blob/3bddc56a1d33707fc2e50674bb2ba678de53fc1b/plugins/plugin-pwa/src/node/injectLinksToHead.ts#L89C64-L89C80

Is there a reason we needed to do this? It violates the WCAG standard to not disable zoom and scaling features of the browser for low vision users.

https://dequeuniversity.com/rules/axe/4.8/meta-viewport?application=AxeChrome
https://www.boia.org/blog/web-accessibility-tips-dont-disable-zooming-yes-even-on-mobile

### Reproduction

https://datagridvue.com/

### Used Package Manager

pnpm

### System Info

```shell
NA
```

**Comment by Mister-Hope** [comment]
An app usually does not allow users to zoom.

But yes, we can add this dynamically when running in a pwa while allowing web pages
